relayer/processor/path_processor.go

```diff
@@ -480,19 +510,19 @@ ChannelHandshakeLoop:
 	}
 }
 
-func (pp *PathProcessor) sendMessages(pathEnd *pathEndRuntime, packetMessages []ibcMessageWithSequence, connectionMessages []ibcMessageWithConnection, channelMessages []ibcMessageWithChannel) error {
-	if len(packetMessages) == 0 && len(connectionMessages) == 0 && len(channelMessages) == 0 {
+func (pp *PathProcessor) sendMessages(pathEnd *pathEndRuntime, messages []provider.RelayerMessage) error {
+	if len(messages) == 0 {
 		return nil
 	}
 	// TODO construct MsgUpdateClient for this pathEnd, using the latest trusted IBC header from other pathEnd, prepend messages with the MsgUpdateClient, then send the messages to this pathEnd
 
-	pp.log.Debug("will send", zap.Any("packet_messages", packetMessages), zap.Any("connection_messages", connectionMessages), zap.Any("channel_messages", channelMessages))
+	pp.log.Debug("will send", zap.Any("messages", messages))
```

Comment:
This looks like some debug logging that could potentially be removed.

Reply:
yes for sure, sending the messages will be in one of the upcoming PRs